test/e2e/accounts/snap-account-eth-swap.spec.ts

```diff
@@ -41,7 +41,6 @@ describe('Snap Account - Swap', function () {
     await withFixtures(
       {
         dappOptions: {
-          defaultTestDapp: 1,
```

Comment:
this is removed as per the bot comment - as it would add an additional server, not needed